.github/workflows/load-soak.yml

```
@@ -0,0 +1,38 @@
+    - cron: '0 2 * * *'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  soak:
+    name: k6 soak (staging)
+    runs-on: ubuntu-latest
+    # A missing secret must never cause a test to fall back to localhost or production.
+    if: ${{ secrets.STAGING_LOAD_API_BASE_URL != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)